### PR TITLE
#160535333-implement-feedback

### PR DESCRIPTION
**Pull request by adex001**
#### What does this PR do?

**1.** Update readme file on github to be comprehensive.
**2.** Include an editor.config file in the application.
**3.** Do not push .env file to the repository. Replace with a dummy
**4.** Updated the HTML document structure and corrects its link.

#### Description of Task to be completed?
N/A

#### What are the relevant pivotal tracker stories?
#160535333

#### Questions:
N/A

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```
Comment (KaiserPhemi) [inline .editorconfig]: #### Nitpick
Always add a new line after the last line

File: UI/HTML/adminsignup.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="../css/style.css">
   <style>@import url('https://fonts.googleapis.com/css?family=Oxygen');
```
Comment (KaiserPhemi) [inline UI/HTML/adminsignup.html]: #### Critical
This should go into the `<link>` tag. No need to create it inside a style tag

File: UI/css/style.css [hunk 3]
```diff
@@ -260,4 +265,20 @@ color: white;
...
+  margin: 20px 80px;
+}
+.checkout{
+  width: 180px;
+  padding: 5px 10px;
+  background: white;
+  color: blue;
+  border: 2px solid greenyellow;
+  margin: 20px 80px;
 }
```
Comment (KaiserPhemi) [inline UI/css/style.css]: #### Nitpick
Always insert a new line after the last line of code

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
Implement feedback on this PR

File: .editorconfig (hunk 1 above)
Comment (adex001) [reply .editorconfig]: this has been effected in all files. Thanks

Comment (adex001) [reply .editorconfig]: it has been effected.  Thanks.

File: UI/HTML/adminsignup.html (hunk 2 above)
Comment (KaiserPhemi) [reply UI/HTML/adminsignup.html]: @adex001 
This is still pending.

#### Critical
Use `<link>` over `@import` as much as possible.
Move this to the `<link>` tag i.e.
```html
<link rel="stylesheet" url="https://.....">
```

**Review by KaiserPhemi — APPROVED** [review]

**Comment by KaiserPhemi** [comment]
# LGTM (Looks Good To Me)

**Pull request merged**